From d1697c5a598082d46762fa85bcfbc2dce224a14a Mon Sep 17 00:00:00 2001
Message-ID: <d1697c5a598082d46762fa85bcfbc2dce224a14a.1726926634.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1726926633.git.sharlatanus@gmail.com>
References: <cover.1726926633.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 18 Sep 2024 15:00:13 +0100
Subject: [PATCH v2 07/25] gnu: python-ginga: Fix tests.

* gnu/packages/astronomy.scm (python-ginga) [arguments]: <#:test-flags>:
Exlude one failing test.

Change-Id: Ic84d89c5cc8cc4938ce8022b1974e5f284179f8e
---
 gnu/packages/astronomy.scm | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 8cb25aab08..980b52e0c0 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1092,6 +1092,8 @@ (define-public python-ginga
     (build-system pyproject-build-system)
     (arguments
      (list
+      ;; AssertionError: Not equal to tolerance rtol=1e-07, atol=0.0001
+      #:test-flags #~(list "-k" "not test_fwhm")
       #:phases
       #~(modify-phases %standard-phases
          (add-after 'unpack 'relax-requirements
-- 
2.46.0

